docs/docs/concepts/overview.mdx

```diff
@@ -0,0 +1,24 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Overview
+
+Compass has three major concept when it comes to data ingestion: Asset, Type, and Service.
+
+Asset is essentially an arbitrary JSON object that represent a metadata of a specific service with a specific type.
+
+Type defines a ‘type’ of an asset and it is pre-defined. There are currently 4 supported types in Compass: `table`, `job`, `dashboard`, and `topic`.
```

Comment:
This is not correct. `feature_table` is also supported. Might be okay to include the types that would be added in #191.